package.json

```diff
@@ -3,13 +3,14 @@
   "version": "0.0.1",
   "description": "",
   "scripts": {
-    "all": "npm run get-all-submodules && npm run cleanup && npm run compile-pos-contracts && npm run make-spec && npm run start-test-setup && npm run test && npm run stop-test-setup",
+    "all": "npm install && npm run get-all-submodules && npm run cleanup && npm run compile-pos-contracts && npm run make-spec && npm run start-test-setup && npm run compile-test && npm run test && npm run stop-test-setup",
     "cleanup": "rm -rf ./parity-data && git checkout ./parity-data",
     "get-all-submodules": "git submodule update --init --remote",
     "compile-pos-contracts": "cd ./pos-contracts && npm i && npm run compile",
     "make-spec": ". ./scripts/network-spec && cd ./pos-contracts && npm i && node ./scripts/make_spec.js",
     "start-test-setup": "bash scripts/start-test-setup ../parity-ethereum/target/debug/parity",
-    "test": "sleep inf",
+    "compile-test": "node_modules/.bin/truffle compile",
+    "test": "node_modules/.bin/truffle test --network node1",
```

Comment:
This needs to be `101`, not `node1`

Reply:
If I'm correctly understanding Truffle networks, each has a dedicated node through which the tests access the network. The Truffle network whose dedicated node is Node 1 is called `node1`.

Reply:
@vkomenda Changing it to `101` fixes the tests for me.  Otherwise, they fail with the same out-of-gas error.